includes/customizer-functions.php

```diff
@@ -21,7 +21,13 @@ function hello_customizer_register( $wp_customize ) {
 		]
 	);
 
-	$wp_customize->add_setting( 'hello-elementor-header-footer', [ 'transport' => 'refresh' ] );
+	$wp_customize->add_setting(
+		'hello-elementor-header-footer',
+		[
+			'sanitize_callback' => 'esc_url',
```

Comment:
Why do you use `esc_url` for header-footer?

Reply:
@KingYes apologies, should have pushed a fix here just running a test on setting sanitize_callback to false (since there is no input)

Will update the PR

Reply:
Unless you recommend one of the other options @KingYes since there's no input to be saved.